cogs/basic.py

```diff
@@ -35,15 +35,18 @@ def __init__(self, bot):
 
     @commands.command(name='xkcd')
     @commands.cooldown(5, 10, commands.BucketType.user)
-    async def get_xkcd(self, ctx):
+    async def get_xkcd(self, ctx, id=None):
         """
         Replies with a random xkcd comic URL.
         :param ctx:
         :return:
         """
-        async with aiohttp.ClientSession() as session:
-            async with session.get('https://c.xkcd.com/random/comic/') as resp:
-                await ctx.send(resp.url)
+        if not id:
```

Comment:
```suggestion
        if not id or id < 1:
```
We can use this check to return a random value if the ID is invalid.